nss-day-cohort-41/nutshell-a-little-budding-rose-by-emily-bronte
# Kc second events update

Added UserId to Event Section

 Type of change
expected)

# Testing Instructions
login to nutshell, add event. check database to make sure the userId to event added matched logged in user

# Checklist:
